src/components/Capture/index.ts

```diff
@@ -26,17 +26,21 @@ export const PoACapture = appendToTracking(
 )
 
 export const SelfieCapture = appendToTracking(
-  withCaptureVariant(Face, {
-    requestedVariant: 'standard',
-    pageId: 'SelfieCapture',
-  }),
+  trackComponent(
```

Comment:
This is not right, please remove it. Which event did you intent to fire?

The events inside `<Face/>` are fired by specific components inside. It's a bit confusing.